backend/store.py

```diff
@@ -68,3 +68,21 @@ def get_random(cls) -> Path:
         files = cls.get_logs()
 
         return Path(random.choice(files))
+
+    @classmethod
+    def get_stats(cls) -> dict:
+        """Retrieve basic statistics about submitted reports.
+        """
+        files = cls.get_logs()
+        stats = {
+            "total_reports" : len(files),
+            "avg_snippets" : 0.0
+        }
+        for path in files:
+            with open(path, encoding='utf-8') as f:
+                report = json.load(f)
```

Comment:
Parsing all reports on each request will get too slow once we have more than a few reports.
I would prefer not parsing the report files and not counting the snippets, only the number of files.

If we want to count snippets, I would at least cache this function